tfgrid-chart/Chart.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: tfgrid-chart
```

please make sure to update it in all files

```suggestion
name: tfgrid-stats
```